domain/infosync/info.go

```diff
@@ -406,6 +413,14 @@ func doRequest(ctx context.Context, addrs []string, route, method string, body i
 			terror.Log(res.Body.Close())
 			return bodyBytes, err
 		}
+		logutil.BgLogger().Warn("fail to doRequest, retry next address",
+			zap.Error(err),
+			zap.String("method", method),
+			zap.String("hosts", addr),
+			zap.String("url", url),
+			zap.Int("http status", res.StatusCode),
```

Comment:
```suggestion
			zap.Int("http status", res.StatusCode),
```

I think the res will be nil here